app/api/speakers_calls.py

```diff
@@ -67,10 +67,14 @@ def before_patch(self, args, kwargs, data):
         if kwargs.get('event_id'):
             try:
                 speakers_call = SpeakersCall.query.filter_by(event_id=kwargs['event_id']).one()
+                event = Event.query.filter_by(id=kwargs['event_id']).one()
```

Comment:
Use ORM links. Don't make redundant queries

Reply:
Used the appropriate linked event .